src/main/java/core/service/impl/TransactionParserImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.service.impl;
+
+import core.model.FruitTransaction;
+import core.service.TransactionParser;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionParserImpl implements TransactionParser {
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_CODE_OPERATION = 0;
+    private static final int INDEX_FRUIT_NAME = 1;
+    private static final int INDEX_FRUIT_QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> parseData(List<String> data) {
+        return data.stream()
+                .map(element -> element.split(SEPARATOR))
+                .map(element -> new FruitTransaction(FruitTransaction.Operation.getOperationByCode(element[INDEX_CODE_OPERATION]),
+                        element[INDEX_FRUIT_NAME], Integer.parseInt(element[INDEX_FRUIT_QUANTITY])))
```

Comment:
getOperationByCode throws IllegalArgumentException, parseInt throws NumberFormatException. Later on, you will have to write tests for fruitshop, so it's probably a good idea to think about handling cases of wrong input format now. Let's catch them and throw an exception informing others that the data has a wrong format